### PR TITLE
Convert false values to empty arrays to prevent boolean error

### PR DESCRIPTION
**Pull request by adamfarley**
Passing "false" in as the value of these variables appears to cause
a Groovy error where it says "No signature of method:
java.lang.Boolean.clear() is applicable for argument types: ()
values: []"

Switching the values to empty arrays, as on the other platforms, to
fix this.

Signed-off-by: Adam Farley <adfarley@redhat.com>

### CODE REVIEW

**Review by sxa — APPROVED** [review]
Seems reasonable, especially since this has been tested, but @gdams should be the second review of this before we merge.

**Review by gdams — APPROVED** [review]

**Review by andrew-m-leonard — APPROVED** [review]

**Comment by adamfarley** [comment]
Thanks everyone. :)

**Pull request merged**
